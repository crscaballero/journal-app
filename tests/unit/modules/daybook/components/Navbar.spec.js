import { shallowMount } from '@vue/test-utils';

import Navbar from '@/modules/daybook/components/Navbar.vue';

describe('Navbar.vue', () => {
    test('It must render the component correctly', () => {
        const wrapper = shallowMount(Navbar);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
