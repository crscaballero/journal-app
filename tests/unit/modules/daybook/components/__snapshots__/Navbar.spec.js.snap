// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar.vue It must render the component correctly 1`] = `
<nav class="navbar bg-primary"><a href="" class="navbar-brand text-white"><img src="" alt="Logo" class="d-inline-block align-text-top mx-2" height="24"> DayBook </a>
  <div class="d-flex"><button class="btn btn-outline-info mx-2"><i class="fa fa-sign-out-alt"></i></button></div>
</nav>
`;
